xrmocap/client/smpl_verts_client.py

```diff
@@ -0,0 +1,181 @@
+# yapf: disable
+import gzip
+import logging
+import numpy as np
+import socketio
+from enum import Enum
+from typing import List, Union
+
+# yapf: enable
+
+
+class XRMocapSMPLClientActionsEnum(str, Enum):
```

Comment:
XRMocap -> XRMoCap?

Reply:
Fixed in [fd72f5b](https://github.com/openxrlab/xrmocap/pull/125/commits/fd72f5b8b53ce9a9e02a76203691d6b4a874f02b). We do not need `xrmocap` in class name because it is a module inside `xrmocap`.